tools/devops/device-tests/cambridge-ios-device-tests.yml

```diff
@@ -0,0 +1,31 @@
+# Runs the device tets on iOS stable devices using the cambridget pools.
+parameters:
+- name: disableProviCleanup
+  type: boolean
+  default: false
+
+- name: clearProviCache
```

Comment:
I like more explicit variables: `clearProvisionatorCache` and `disableProvisionatorCache`. It's more to type, but it's also easier to understand for new people reading this for the first time.